plugins/dependencies/src/main/kotlin/de/fayard/refreshVersions/RefreshVersionsExtension.kt

```diff
@@ -8,6 +8,11 @@ open class RefreshVersionsExtension {
 
     var versionsPropertiesFile: File? = null
     var extraArtifactVersionKeyRules: List<String> = emptyList()
+    var isBuildSrcLibsEnabled = false
```

Comment:
Do we need to expose this variable in the DSL? `enableBuildSrcLibs` seems enough to me, I don't see the point of configuring `enableBuildSrcLibs = false`

Reply:
I can make it internal but cannot remove it entirely since the design of Gradle extensions require to have a backing field to be able to access what's been configured layer on.

Reply:
Made `isBuildSrcLibsEnabled` internal.